local-cli/runAndroid/runAndroid.js

```diff
@@ -150,9 +152,9 @@ function tryInstallAppOnDevice(args, device) {
   }
 }
 
-function tryLaunchAppOnDevice(device, packageName, adbPath, mainActivity) {
+function tryLaunchAppOnDevice(device, packageNameWithSuffix, packageName, adbPath, mainActivity) {
   try {
-    const adbArgs = ['-s', device, 'shell', 'am', 'start', '-n', packageName + '/.' + mainActivity];
+    const adbArgs = ['-s', device, 'shell', 'am', 'start', '-n', packageNameWithSuffix + '/' + packageName +  + mainActivity];
```

Comment:
@Phredward This be `+ '.' + mainActivity` right? With the current implementation it breaks and searches for `≤packageName>NaN` activity. cc: @ericvicenti